src/vizceral.jsx

```diff
@@ -200,18 +217,18 @@ Vizceral.defaultProps = {
   connectionHighlighted: () => {},
   definitions: {},
   filters: [],
-  graphsUpdated: () => {},
+  graphsUpdated: () => undefined,
```

Comment:
Why the change? These have default empty functions so they are just no-ops if the user does not provide listeners to things they do not care about.